corral/main.pony

```diff
@@ -31,6 +34,8 @@ actor Main
             default' = false)
         ],
         [
+          CommandSpec.leaf(
+            "version", "Show the version and exit")?
```

Comment:
The surrounding code breaks the second argument to the next line. We should remain consistent.

Reply:
Argh, my bad :pray: